events/lastAvatarStandingTrivia/clientScripts/triviaMasterClientStandard.js

```diff
@@ -0,0 +1,747 @@
+                            break;
+                        case "Trivia Timer Countdown":
+                            timer = element;
+                            break;
+                        case "Trivia Avatar Counter Total":
+                            avatarCounter = element;
+                            break;
+                        case "Trivia Player Game Zone":
+                            gameZone = element;
+                            gameZoneProperties = Entities.getEntityProperties(gameZone, 
+                                ["id", "position", "dimensions", "rotation", "userData"]);
+                            break;
+                        case "Trivia Prize Amount":
+                            prizeDisplay = element;
+                            break;                            
+                        case "Trivia Bubble":
+                            bubble = element;
+                            break;
+                    }
+                }
+            }
+        });
+    }
+
+    function onClicked() { 
+        if (open) {
+            tablet.gotoHomeScreen();
+        } else {
+            tablet.gotoWebScreen(appPage);
+        }
+    }
+
+    // function getQuestion() {
+    //     Entities.callEntityServerMethod(gameZoneProperties.id, "playSound", ['NEXT_QUESTION_SFX']);
+    //     try {
+    //         var triviaURL = SECRETS.trivia_URL,
+            
+    //         // if (type) {
+    //         //     triviaURL = triviaURL + "&type=" + type;
+    //         // }
+    //         // if (difficulty) {
+    //         //     triviaURL = triviaURL + "&difficulty=" + difficulty;
+    //         // }
+    //         // if (category) {
+    //         //     triviaURL = triviaURL + "&category=" + category;
+    //         // }
+    //         triviaURL = triviaURL + "category=" + "Giraffage_Hemsworth";
+    //         request(triviaURL, function (error, data) {
+    //             if (!error) {
+    //                 console.log(JSON.stringify(data));
+    //                 tablet.emitScriptEvent(JSON.stringify(data));
+    //                 triviaData = data;
+    //             }
+    //         });
+    //     } catch (err) {
+    //         console.log("err:", err);
+    //         print("Could not get domain data using userData domainAPIURL");
+    //     }
+    // }
+
+    function getQuestion() {
+        Entities.callEntityServerMethod(gameZoneProperties.id, "playSound", ['NEXT_QUESTION_SFX']);
+        try {
+            var triviaURL = "https://opentdb.com/api.php?amount=1";
+            if (type) {
+                triviaURL = triviaURL + "&type=" + type;
+            }
+            if (difficulty) {
+                triviaURL = triviaURL + "&difficulty=" + difficulty;
+            }
+            if (category) {
+                triviaURL = triviaURL + "&category=" + category;
+            }
+            request(triviaURL, function (error, data) {
+                if (!error) {
```

Comment:
what happens if there _is_ an error here?

Reply:
This will show an error on the tablet, most commonly we've seen this for custom databases where it was possible to run through all the questions and once used they were marked to avoid reuse.  It has not happened with the openTriviaDB.

Reply:
how will this show an error on the tablet? i don't see the code in here for displaying any errors

Reply:
I've updated the function to send the tablet an 'error' string.  On emitScriptEvent, the tablet will alert the host to check the question database (in html line 112).